### PR TITLE
Update proxy SHA to almost latest.

### PR DESCRIPTION
**Pull request by qiwzhang**
Reversed MixerClient stats change which causes seg fault.

### CODE REVIEW

File: mixer/test/client/network_policy/network_policy_test.go [hunk 1]
```diff
@@ -78,7 +78,7 @@ func TestNetworkFailure(t *testing.T) {
 	if err != nil {
 		t.Errorf("Failed in request %s: %v", tag, err)
 	}
-	if code != 503 {
-		t.Errorf("Status code 503 is expected, got %d.", code)
+	if code == 200 {
+		t.Errorf("Non-200 status code 503 is expected, got %d.", code)
```
Comment (douglas-reid) [inline mixer/test/client/network_policy/network_policy_test.go]: curious: what changed re: status code in the update?

Also, you should remove "code 503" from the error message now.

Comment (qiwzhang) [reply mixer/test/client/network_policy/network_policy_test.go]: Thanks.  Removed it.

Envoy gRPC implementation has changed.   If gRPC server is not up,  it used to return status_code=LOSS_DATA.  Now it returns CACNCELED.

**Comment by ldemailly** [comment]
🎉

**Pull request merged**